--- a/src/utils.h
+++ b/src/utils.h
@@ -79,8 +79,8 @@
 int buffer_write_to_filename(const char *filename, const char *buffer, uint64_t length);
 
 enum plist_format_t {
-	PLIST_FORMAT_XML,
-	PLIST_FORMAT_BINARY
+	USBMUXD_PLIST_FORMAT_XML,
+	USBMUXD_PLIST_FORMAT_BINARY
 };
 
 int plist_read_from_filename(plist_t *plist, const char *filename);
--- a/src/utils.c
+++ b/src/utils.c
@@ -310,9 +310,9 @@
 	if (!plist || !filename)
 		return 0;
 
-	if (format == PLIST_FORMAT_XML)
+	if (format == USBMUXD_PLIST_FORMAT_XML)
 		plist_to_xml(plist, &buffer, &length);
-	else if (format == PLIST_FORMAT_BINARY)
+	else if (format == USBMUXD_PLIST_FORMAT_BINARY)
 		plist_to_bin(plist, &buffer, &length);
 	else
 		return 0;
--- a/src/conf.c
+++ b/src/conf.c
@@ -254,7 +254,7 @@
 		usbmuxd_log(LL_DEBUG, "Setting key %s in config file %s", key, config_file);
 	}
 
-	int res = plist_write_to_filename(config, config_file, PLIST_FORMAT_XML);
+	int res = plist_write_to_filename(config, config_file, USBMUXD_PLIST_FORMAT_XML);
 
 	plist_free(config);
 
@@ -428,7 +428,7 @@
 	remove(device_record_file);
 
 	/* store file */
-	if (!plist_write_to_filename(plist, device_record_file, PLIST_FORMAT_XML)) {
+	if (!plist_write_to_filename(plist, device_record_file, USBMUXD_PLIST_FORMAT_XML)) {
 		usbmuxd_log(LL_DEBUG, "Could not open '%s' for writing: %s", device_record_file, strerror(errno));
 		res = -ENOENT;
 	}
